src/views/results/resultSetPanelProvider.ts

```diff
@@ -41,9 +41,12 @@ export class ResultSetPanelProvider implements WebviewViewProvider {
 
           let queryResults = queryObject.getState() == QueryState.RUN_MORE_DATA_AVAILABLE ? await queryObject.fetchMore() : await queryObject.run();
 
+          const jobId = queryObject.getHostJob().id;
+
           data = queryResults.data;
```

Comment:
Doesn't look like we need `data`.  It's defined on line 29 and set here, but is never used.